docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -377,6 +377,71 @@ def repo():
 
 Like regular jobs, asset jobs can be placed on [schedules](/concepts/partitions-schedules-sensors/schedules) and [sensors](/concepts/partitions-schedules-sensors/sensors).
 
+---
+
+## Grouping assets
+
+To help keep your assets tidy, you can organize them into groups. Grouping assets by project, concept, and so on simplifies [keeping track of them in Dagit](#viewing-asset-groups-in-dagit).
+
+- [Assigning assets to groups](#assigning-assets-to-groups)
+- [Viewing asset groups in Dagit](#viewing-asset-groups-in-dagit)
+
+### Assigning assets to groups
+
+In Dagster, there are two ways to assign assets to groups:
+
+- [By using the `load_assets_from_package_module` function](#from-assets-in-a-sub-module) (**recommended**)
```

Comment:
I would slightly reword this, as it makes it sound a bit like the purpose of `load_assets_from_package_module` is to apply a group to those assets. Maybe:

"By using the group_name argument when calling `load_assets_from_package_module` or `load_assets_from_current_module`"